src/NServiceBus.Core/Routing/FileBasedDynamicRouting/FileRoutingTable.cs

```diff
@@ -47,6 +50,50 @@ public void ReloadData()
             }
         }
 
+        static void LogChanges(Dictionary<string, HashSet<EndpointInstance>> oldInstanceMap, Dictionary<string, HashSet<EndpointInstance>> newInstanceMap)
+        {
+            var output = new StringBuilder();
+            var hasChanges = false;
+            output.AppendLine("Updating instance mapping table:");
```

Comment:
Would it make sense to output `Updating instance mapping table from {absoluteFilePath}:` ?